apps/website/src/entries.tsx

```diff
@@ -74,36 +88,34 @@ export default createPages(async ({ createPage, createLayout }) => {
   // Initialise a map for the homepages, since we want to exclude them from
   // creating a page for their internal path.
   const homePages = await query(HomePageQuery, {});
-  const homePageTranslations = new Map<Locale, Url>();
+  const homePageTranslations = [] as Array<Url>;
```

Comment:
Why are we using `as` here?

```suggestion
  const homePageTranslations: Array<Url> = [];
```